beaconclient/prod_beacon_instance.go

```diff
@@ -58,6 +79,30 @@ func (c *ProdBeaconInstance) SubscribeToHeadEvents(slotC chan HeadEventData) {
 	}
 }
 
+func (c *ProdBeaconInstance) SubscribeToPayloadAttributesEvents(payloadAttributesC chan PayloadAttributesData) {
+	eventsURL := fmt.Sprintf("%s/eth/v1/events?topics=payload_attributes", c.beaconURI)
+	log := c.log.WithField("url", eventsURL)
+	log.Info("subscribing to payload_attributes events")
+
+	for {
+		client := sse.NewClient(eventsURL)
```

Comment:
would there be any advantage in reusing the existing SSE client across subscriptions?